lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
The extra migration suggestion seems viable, I'd vote for that option.

https://github.com/edx/edx-platform/pull/13455